### PR TITLE
JCS-211 - Use Map.forEach()

### PR DESCRIPTION
**Pull request by arturobernalg**

### CODE REVIEW

**Comment by tvand** [comment]
Hi Arturo,

would you please stick to the code formatting conventions of the project?

Bye, Thomas.

File: commons-jcs-core/src/main/java/org/apache/commons/jcs3/engine/CacheWatchRepairable.java [hunk 1]
```diff
@@ -58,22 +58,16 @@
...
-                    cacheWatch.addCacheListener( cacheName, listener );
-                }
-                catch ( final IOException ex )
-                {
-                    log.error( "Problem adding listener. ICacheListener = {0} | "
-                            + "ICacheObserver = {1}", listener, cacheWatch, ex );
-                }
-            });
-        });
+        cacheMap.forEach((cacheName, value) -> value.forEach(listener -> {
```
Comment (tvand) [inline commons-jcs-core/src/main/java/org/apache/commons/jcs3/engine/CacheWatchRepairable.java:61]: Please use the code formatting conventions of the project.

**Review by tvand — CHANGES_REQUESTED** [review]

File: commons-jcs-core/src/main/java/org/apache/commons/jcs3/engine/CacheWatchRepairable.java (hunk 1 above)
Comment (arturobernalg) [reply commons-jcs-core/src/main/java/org/apache/commons/jcs3/engine/CacheWatchRepairable.java:61]: HI @tvand 
I think it's solved.
TY

**Review by tvand — APPROVED** [review]
Thank you.

**Pull request merged**